WalletWasabi.Fluent/Views/Wallets/MusicBoxCoinjoinProfileSelector.axaml

```diff
@@ -0,0 +1,163 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:profiles="using:WalletWasabi.Fluent.ViewModels.CoinJoinProfiles"
+             xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.MusicBoxCoinjoinProfileSelector">
+  <UserControl.Styles>
+    <Style Selector="ToolTip">
+      <Setter Property="MaxWidth" Value="1000" />
+    </Style>
+    <Style Selector="FlyoutPresenter">
+      <Setter Property="ScrollViewer.VerticalScrollBarVisibility" Value="Disabled" />
+      <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled" />
+      <Setter Property="Height" Value="580" />
+      <Setter Property="MaxHeight" Value="580" />
+    </Style>
+
+    <Style Selector="FlyoutPresenter > Panel">
+      <Setter Property="Height" Value="520" />
+      <Setter Property="MaxHeight" Value="520" />
+    </Style>
+
+    <Style Selector="c|TileButton">
+      <Setter Property="Margin" Value="10" />
+      <Setter Property="Padding" Value="0" />
+      <Setter Property="Width" Value="300" />
+      <Setter Property="Height" Value="90" />
+      <Setter Property="Foreground" Value="{DynamicResource TextForegroundColor}" />
+      <Setter Property="VerticalContentAlignment" Value="Stretch" />
+      <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+      <Setter Property="BorderBrush" Value="Transparent" />
+      <Setter Property="BorderThickness" Value="1" />
+      <Setter Property="Command" Value="{Binding $parent[UserControl].DataContext.SelectProfileCommand}" />
+      <Setter Property="CommandParameter" Value="{Binding}" />
+    </Style>
+    <Style Selector="c|TileButton.selected">
+      <Setter Property="BorderBrush" Value="{DynamicResource SystemAccentColor}" />
+      <Setter Property="Foreground" Value="{DynamicResource SystemAccentColor}" />
+    </Style>
+  </UserControl.Styles>
+  <Button Classes="plain"
+          Command="{Binding OpenFlyoutCommand}">
+    <Interaction.Behaviors>
+      <behaviors:BindableFlyoutOpenBehavior IsOpen="{Binding IsFlyoutOpen}" AutoOpenOnPointerOver="False" />
+    </Interaction.Behaviors>
+
+    <Panel ToolTip.Tip="{Binding SelectedProfile.Title, StringFormat='Selected Coinjoin Profile: {0} (click to change)', FallbackValue='Click to select your Coinjoin Profile'}"
```

Comment:
I would remove `click to change` the hand icon when hovering should be enough IMO.

```suggestion
    <Panel ToolTip.Tip="{Binding SelectedProfile.Title, StringFormat='Coinjoin Profile: {0}', FallbackValue='Click to select your Coinjoin Profile'}"
```